internal/driver/glfw/window.go

```diff
@@ -838,6 +838,8 @@ func (w *window) mouseScrolled(viewport *glfw.Window, xoff float64, yoff float64
 		}
 		ev := &fyne.ScrollEvent{}
 		ev.Scrolled = fyne.NewDelta(float32(xoff)*scrollSpeed, float32(yoff)*scrollSpeed)
+		ev.Position = pos
+		ev.AbsolutePosition = w.mousePos
```

Comment:
Could you add a TODO comment here? Just to be aware that when this is merged with the develop branch, it needs to be updated with locks. 🤔

Reply:
I don't like to add TODO in PR if it can be avoided. I have written it on my merge list so it's addressed as soon as the release merges

Reply:
Great! :)